micropub.php

```diff
@@ -271,7 +278,14 @@ private static function generate_args() {
         $args[$mp_to_wp[$param]] = $value;
       }
     }
-
+    // if a title is set but not a slug set slug to title
```

Comment:
minor: these two conditionals are pretty self-documenting, no need for the comments